docs/IntegrationTest.md

```diff
@@ -12,17 +12,62 @@
  License.
 -->
 # Integration Tests
+## Setup
+1. Make sure you have another cluster and service account set up for testing deployments as described in the GKE Plugin [usage documentation](Home.md#usage).
 
-## Testing the plugin on Jenkins
+1. Repeat steps in the [IAM Credentials](Home.md#iam-credentials) section on this new Jenkins instance with a new GCP service account.
+
+1. Create the following cluster role:
```

Comment:
Let's avoid duplicating documentation, and just use a link to the appropriate section in the Home.md file.

Reply:
Do you mean the cluster role part that's a duplication? It's a role binding for a non-default namespace "test".

If it's the IAM credentials portion, are you referring to it being repeated in both integration testing and testing the plugin in Jenkins section?

Reply:
Below as well.

Reply:
The role definition exists outside of any namespace. The YAML for its creation doesn't need to be duplicated.

Reply:
Also the IAM portion is exactly the same as well.

Reply:
Done.